db/order_puppies.js

```diff
@@ -0,0 +1,92 @@
+const client = require("./client");
+
+async function addToOrderPuppies({
+    userId,
+    puppyId
+}) {
+    try{
+        const { rows: [order_puppy] } = await client.query(`
+            INSERT INTO order_puppies("userId", "puppyId")
+            VALUES ($1, $2)
+            ON CONFLICT ("userId", "puppyId") DO NOTHING
+            RETURNING *;
+        `, [userId, puppyId])
+
+        return order_puppy;
+    } catch (error) {
+        console.error(error)
+    }
+}
+
+async function getOrderPuppyById(id) {
```

Comment:
I think this function needs a JOIN with the puppies table to get the puppy info we want and we probably only need to grab the puppy id, name and price columns since I don't think we need/want to display too much puppy info on the order. We'll probably just include a link to the individual puppy page or something.